src/main/java/spark/webserver/jetty/JettyServerFactory.java

```diff
@@ -22,27 +22,29 @@
 /**
  * Creates Jetty Server instances.
  */
-public class JettyServerFactory {
+class JettyServerFactory {
```

Comment:
rename to `JettyServer`??? This way where used it would be `JettyServer.create` which is very describing.